packages/docs/pages/Datepicker/DatepickerPage.tsx

```diff
@@ -0,0 +1,85 @@
+import { Datepicker, Form, FormGroup, H0, H1, Text } from '@bigcommerce/big-design';
+import React, { useState } from 'react';
+
+import { Code, CodePreview } from '../../components';
+import { DatepickerPropTable } from '../../PropTables';
+
+const DatepickerPage = () => (
+  <>
+    <H0>Datepicker</H0>
+
+    <Text>Datepicker is good</Text>
+
+    <CodePreview>
+      {/* jsx-to-string:start */}
+      {function Example() {
+        const [date, setDate] = useState<string>();
+
+        return (
+          <Form>
+            <FormGroup>
+              <Datepicker
+                label="Label"
+                placeholder="Placeholder"
+                min="06/03/2020"
+                max="06/19/2020"
+                onDateChange={(value: string) => setDate(value)}
+                value={date}
+                locale="en-AU"
+              />
+            </FormGroup>
+          </Form>
+        );
+      }}
+      {/* jsx-to-string:end */}
+    </CodePreview>
+
+    <H1>API</H1>
+
```

Comment:
We should also add this:
```suggestion
    <H1>API</H1>
    
    <Text>
      Supports all native <Code>&lt;input [type="date"] /&gt;</Code> element attributes.
    </Text>

```